src/library/Uno.Material/Styles/Controls/v2/CommandBar.xaml

```diff
@@ -24,145 +25,28 @@
 		<Setter Property="Template">
 			<Setter.Value>
 				<ControlTemplate TargetType="AppBarButton">
-					<um:Ripple Feedback="{TemplateBinding Foreground}"
-									 FeedbackOpacity="{StaticResource PressedOpacity}">
-						<Viewbox x:Name="ContentViewbox"
-								 MaxHeight="{ThemeResource AppBarButtonContentHeight}"
-								 MaxWidth="{ThemeResource AppBarButtonContentHeight}"
-								 Margin="0,12"
-								 HorizontalAlignment="Stretch"
-								 StretchDirection="DownOnly"
-								 AutomationProperties.AccessibilityView="Raw">
-							<ContentPresenter x:Name="Content"
-											  Content="{TemplateBinding Icon}"
-											  Foreground="{TemplateBinding Foreground}" />
+					<um:Ripple Feedback="{TemplateBinding Foreground}" FeedbackOpacity="{StaticResource PressedOpacity}">
+						<Viewbox x:Name="ContentViewbox" MaxHeight="{ThemeResource AppBarButtonContentHeight}" MaxWidth="{ThemeResource AppBarButtonContentHeight}" Margin="0,12" HorizontalAlignment="Stretch" StretchDirection="DownOnly" AutomationProperties.AccessibilityView="Raw">
```

Comment:
![image](https://github.com/unoplatform/Uno.Themes/assets/2359550/c0747273-5d36-48eb-97a4-ef2ba6c8695e)

Reply:
thanks, i've updated my settings

Reply:
please take a look at the diff set
your xamlstyler is not setup as expected

Reply:
![image](https://github.com/unoplatform/Uno.Themes/assets/4793020/aa33c778-b22d-4388-8662-d2b88e23af1f)


Make sure you are using tabs/mixed as the indentation style. I see spaces being used here

Reply:
yes, currently fixing my styler setting.